clientv3/op.go

```diff
@@ -553,8 +553,8 @@ func toLeaseTimeToLiveRequest(id LeaseID, opts ...LeaseOption) *pb.LeaseTimeToLi
 	return &pb.LeaseTimeToLiveRequest{ID: int64(id), Keys: ret.attachedKeys}
 }
 
-// isWithPrefix returns true if WithPrefix is being called in the op
-func isWithPrefix(opts []OpOption) bool { return isOpFuncCalled("WithPrefix", opts) }
+// IsWithPrefix returns true if WithPrefix option is called in the given opts.
```

Comment:
While you are working on the test, can you change `IsWithPrefix` to `IsOptsWithPrefix`?

Reply:
I got it.